src/Makefile.am

```diff
@@ -73,6 +75,8 @@ libproj_la_SOURCES = \
 	geocent.c geocent.h pj_utils.c pj_gridinfo.c pj_gridlist.c \
 	jniproj.c pj_mutex.c pj_initcache.c pj_apply_vgridshift.c geodesic.c \
 	pj_strtod.c
+	\
```

Comment:
This line continuation doesn't make sense here. All it does is join the next line with this one, so you've just placed a couple file names here all by themselves.

The backslash needs to be on the previous line, or you need to just put the file names there directly.

I don't know why Travis doesn't just give up earlier, though.